contrib/gdk-pixbuf/loader.c

```diff
@@ -423,7 +422,7 @@ static gboolean avif_image_saver(FILE          *f,
         }
     }
 
-    max_quantizer = AVIF_QUANTIZER_WORST_QUALITY * ( 100 - CLAMP(quality, 0, 100)) / 100;
+    max_quantizer = AVIF_QUANTIZER_WORST_QUALITY * (100 - (int)quality) / 100;
```

Comment:
The `CLAMP(quality, 0, 100)` call is not necessary because of the check at line 367.